src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1374,7 +1374,7 @@ components:
         emitter:
           type: object
         data:
-          type: object
```

Comment:
Just a reminder (for want of a better place to write it) to add a description to the entity state endpoint explaining that eg the user may wish to cast to ProgrammaticScryptoSborValue.